src/community/geopkg/src/main/java/org/geoserver/geopkg/wps/GeoPackageProcess.java

```diff
@@ -314,6 +314,7 @@ else if (tiles.getSldBody() != null) {
                }
                request.setFormat("none");
                Map formatOptions = new HashMap();
+               formatOptions.put("flipy", "true");
```

Comment:
I am wondering if we are fixing this issue (https://osgeo-org.atlassian.net/browse/GEOT-5582) and then running into trouble downstream here in geoserver

Reply:
I don't think so. It isn't CRS dependent, more about how tile indices (in a TMS z/x/y sense) are used. GeoPackage does it one way, mbtiles the other.